code/renderers/vue3/src/render.ts

```diff
@@ -145,3 +149,9 @@ function teardown(
   storybookApp?.unmount();
   if (map.has(canvasElement)) map.delete(canvasElement);
 }
+
+async function installGlobalPlugins(app: App<any>, storyContext: StoryContext<VueRenderer>) {
+  if (window.APPLY_PLUGINS_FUNC) {
+    await window.APPLY_PLUGINS_FUNC(app, storyContext);
```

Comment:
What is the difference with the setup function? Should this function be set by the user?

Reply:
using setup is not safe, in case your package manager allows 2 versions of the same package, @storybook/vue3 can't be a singleton,  so calling setup. seems not working for users 
using window. APPLY_PLUGINS_FUNC  will insure the singleton. 
it should be. awaited as well in case. registering the plugin is async

Reply:
I think we can also solve this by putting this on the window:
https://github.com/storybookjs/storybook/blob/936602d1bba26788298d3aa839e38ddb248c4fbc/code/renderers/vue3/src/render.ts#L20

I think it is better have one API.

Reply:
yes exactly i'm working on that idea